src/test/java/io/stargate/sgv2/jsonapi/testresource/SerialConsistencyOverrideProfile.java

```diff
@@ -0,0 +1,19 @@
+package io.stargate.sgv2.jsonapi.testresource;
+
+import com.google.common.collect.ImmutableMap;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class SerialConsistencyOverrideProfile implements QuarkusTestProfile {
```

Comment:
nit: just move this as inner class of the test `SerialConsistencyOverrideOperationTest.java`, it's way easier to see immediately what the profile is.. That's how we did it so far, very easy to understand if you see the profile at the top of the test class immediatelly..

Reply:
Make sense, done.